compiler/p/codegen/PPCOpsDefines.hpp

```diff
@@ -255,6 +261,16 @@ FORMAT_BF_RA_RB,
 // +------+-----+----+----------+----------+----------------------+
 FORMAT_BF_FRA_FRB,
 
+// Format for instructions with an FRT field encoding the target FP register and RA and RB fields
+// encoding the source registers:
+//
+// +------+----------+----------+----------+----------------------+
+// |      | FRT      | RA       | RB       |                      |
+// | 0    | 6        | 11       | 16       | 21                   |
+// +------+----------+----------+----------+----------------------+
+FORMAT_FRT_RA_RB,
```

Comment:
is there a non-memory-oriented instruction using RA, RB, and using or defining FPR/VR/VSR?

Reply:
At present, there are not any instructions using this (or the same formats with `VRT`/`XT`). These were simply added for completeness's sake. The only cost of adding these formats is taking up a couple of additional values in this enum, there's no extra code needed for actually handling these formats above and beyond what's already there for handling the memory variants of the same formats.

Also, just to preempt questions on this topic and prevent any confusion: the `*RS` variants did not get the same treatment since there's no instruction class that could reasonably be used to encode them at present. Such an instruction would theoretically use a `PPCSrc3Instruction` class, but no such class exists right now and adding one is beyond the scope of this PR.

Reply:
the better answer is:  there has never been this kind of instructions, and don't expect them in the future in general, since it is architecturally not good (pinching hardware implementation, that is).

PPCSrc3Instruction (without a memory component) doesn't make sense intuitively either, unless it is an interim class.  We have enough nop(s) already.

Reply:
These formats have been removed in 497d4c96ec4afe9492e3f8b13c0aeb5f696cf853.